catalyst/contrib/dl/runner/wandb.py

```diff
@@ -129,8 +131,12 @@ def _run_epoch(self, stage: str, epoch: int):
             for mode, metrics in \
                     self.state.metric_manager.epoch_values.items():
                 self._log_metrics(
-                    metrics=metrics, mode=mode, suffix=self.epoch_log_suffix
+                    metrics=metrics,
+                    mode=mode,
+                    suffix=self.epoch_log_suffix,
+                    epoch=epoch
                 )
+            wandb.log(commit=True)
```

Comment:
why do we need to do this?

Reply:
From WandB documentation:
> As long as you keep passing the same value for `step`, W&B will collect the keys and values from each call in one unified dictionary. As soon you call `wandb.log()` with a different value for `step` than the previous one, W&B will write all the collected keys and values to the history, and start collection over again.

This means that if there's no this line:
1. Results of epoch `i` are sent to WandB when you already obtained results for epoch `i + 1`
2. Results of last epoch are not sent because you won't make `wandb.log()` with different step.